subprojects/core-api/src/main/java/org/gradle/api/artifacts/ResolutionStrategy.java

```diff
@@ -27,7 +27,7 @@
  * For example, forcing certain dependency versions, substitutions, conflict resolutions or snapshot timeouts.
  * <p>
  * Examples:
- * <pre class='autoTested'>
```

Comment:
And this?

Reply:
Same reason. The substituted project does not exist in the test project.